src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -13184,6 +13184,121 @@ class D : C
             Assert.True(m2.OverriddenMethod.Parameters[0].Type.IsNullableType());
         }
 
+        [Fact]
+        [WorkItem(33276, "https://github.com/dotnet/roslyn/issues/33276")]
+        public void Overriding_34()
+        {
+            var source1 =
+@"
+public class MyEntity
+{
+}
+
+public abstract class BaseController<T>
+    where T : MyEntity
+{
+    public abstract void SomeMethod<R>(R? lite)
+        where R : MyEntity;
+}
+";
+
+            var source2 =
+@"
+class DerivedController<T> : BaseController<T>
+    where T : MyEntity
+{
+    Table<T> table = null!;
+    public override void SomeMethod<R>(R? lite) where R : class
```

Comment:
>where R : class [](start = 48, length = 15)

is it also worth testing without the constraint on the override (as originally reported in the bug)?

As a side note, was there a discussion to port the fix for compat break to 16.0 (the fix is currently made it in 16.1p3)?